src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintDocumentService.java

```diff
@@ -44,7 +44,7 @@ public void validateEncryptionOnUploadedDocument(CaseDocument caseDocument,
             String documentFilename = caseDocument.getDocumentFilename();
             log.info("checking encryption for file {} for Case ID: {}", documentFilename, caseId);
             byte[] pdfBytes;
-            if (documentFilename.endsWith(".doc") || documentFilename.endsWith(".docx")) {
+            if (documentFilename.toLowerCase().endsWith(".doc") || documentFilename.toLowerCase().endsWith(".docx")) {
```

Comment:
It's better to check null
```
Optional.ofNullable(documentFilename)
            .map(String::toLowerCase)
            .map(name -> name.endsWith(".doc") || name.endsWith(".docx"))
            .orElse(false)
            ```

Reply:
It is better but in this case filename is unlikely to be null since the code is called from the upload form where there is bound to be a filename otherwise the front-end validation will prevent the form being submitted